### PR TITLE
Adding more content to About.jsx. Trying to add images now.

### PR DESCRIPTION
**Pull request by funbunch**
This refers to issue #1202. 

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

**Review by nichhk — CHANGES_REQUESTED** [review]
Hey Shannan, thanks for working on this! I think the issue is that `mui-image` is only [compatible](https://github.com/benmneb/mui-image#3-profit-) with material-ui v5. So we need to use the older `material-ui-image` package.

**Comment by funbunch** [comment]
Ohh. Will try that!

**Comment by funbunch** [comment]
Latest update so far with images showing now. 

Screen Shot 2022-08-15 at 5 00 34 PM

File: client/components/main/About.jsx [hunk 1]
```diff
@@ -39,26 +33,62 @@ const useStyles = makeStyles({
...
+              using an easy-to-use application, EmpowerLA partnered with Hack
+              For LA  to create the 311 Data project. The 311 Data project makes
+              navigating the wealth of 311 data easier using an open source
+              application built and maintained by volunteers throughout our community.
+            </p>
+          </Grid>
+        </Grid>
+        <Grid container>
+          <Grid item xs={12} sm={3}>
+            <Image src={partner1} animationDuration={0} width={378} />
```
Comment (nichhk) [inline client/components/main/About.jsx:82]: Is it necessary to give the `animationDuration` arg?

**Review by nichhk — CHANGES_REQUESTED** [review]
Looks great! Cool to see the Grid in action.

For the images, please give them more descriptive names. the first one could be `empower_la_logo.png`. Please keep filenames lower case as well.

You mentioned the image sizing issue--let me know if you run into problems with that. Great progress though!

File: client/components/main/About.jsx (hunk 1 above)
Comment (funbunch) [reply client/components/main/About.jsx:82]: Great, will make changes! And test if `animationDuration` is needed.

**Comment by funbunch** [comment]
Screen Shot 2022-08-21 at 8 39 35 PM

It seemed the MUI Image component was more for an image gallery so I get it to work with the tag img.

File: client/components/main/About.jsx [hunk 2]
```diff
@@ -4,10 +4,9 @@ import {
   Container,
   Grid,
 } from '@material-ui/core';
-import Image from 'material-ui-image';
-import partner1 from '@assets/Partners_icon_1.png';
-import partner2 from '@assets/Partners_icon_2.png';
-import partner3 from '@assets/Partners_icon_3.png';
+import partner1 from '@assets/empower_la_logo.png';
```
Comment (nichhk) [inline client/components/main/About.jsx:7]: nit: let's give the imports more meaningful names as well, e.g.,:
`import empower_la_logo from '@assets/empower_la_logo.png';`

File: client/components/main/About.jsx [hunk 3]
```diff
@@ -4,10 +4,9 @@ import {
...
+import partner2 from '@assets/hackforla_logo.png';
```
Comment (nichhk) [inline client/components/main/About.jsx:8]: nit: add underscores to filename. i.e., hack_for_la_logo.png.

File: client/components/main/About.jsx [hunk 4]
```diff
@@ -4,10 +4,9 @@ import {
...
+import partner3 from '@assets/codeforamerica.png';
```
Comment (nichhk) [inline client/components/main/About.jsx:9]: nit: code_for_america_logo.png

File: client/package.json [hunk 5]
```diff
@@ -32,6 +32,7 @@
     "html2canvas": "^1.0.0-rc.5",
     "jest": "^24.9.0",
     "mapbox-gl": "^1.12.0",
+    "material-ui-image": "^3.3.2",
```
Comment (nichhk) [inline client/package.json:35]: Since we're not using this anymore, please remove it.

**Review by nichhk — CHANGES_REQUESTED** [review]
Looks great Shannan! Just a few small comments, and then we can merge this PR.

File: client/package.json (hunk 5 above)
Comment (funbunch) [reply client/package.json]: Latest capture. Updated naming and minor style changes. 

Screen Shot 2022-08-22 at 10 23 57 PM

**Review by nichhk — APPROVED** [review]
LGTM, thanks Shannan! Please merge this at your convenience.

**Pull request merged**